libarchive/archive_write_set_format_pax.c

```diff
@@ -329,17 +329,42 @@ add_pax_attr_binary(struct archive_string *as, const char *key,
 	archive_strappend_char(as, '\n');
 }
 
+static void
+archive_write_pax_header_xattr(struct pax *pax, const char *encoded_name,
+    const void *value, size_t value_len)
+{
+	struct archive_string s;
+	char *encoded_value;
+
+	if (1) {
```

Comment:
It's not a test we need - it's an API, right?  I don't think we can really key off anything in the format, unless we were willing to change libarchive to unconditionally switch to emitting `SCHILY` for xattrs.

Although actually, `archive_write_set_format_gnutar` should certainly switch I'd say.